frontend/packages/process-editor/src/components/Canvas/BPMNEditor/BPMNEditor.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BPMNEditor } from './BPMNEditor';
+import { BpmnApiContextProvider } from '../../../contexts/BpmnApiContext';
+import { BpmnConfigPanelFormContextProvider } from '../../../contexts/BpmnConfigPanelContext';
+import { textMock } from '../../../../../../testing/mocks/i18nMock';
+
+jest.mock('../../../hooks/useBpmnEditor', () => ({
+  useBpmnEditor: jest.fn().mockReturnValue({
+    canvasRef: { current: document.createElement('div') },
+  }),
+}));
+
+describe('BPMNEditor', () => {
+  it('render spinner when pendingApiOperations is true', () => {
```

Comment:
Could it be nice to add this to the top of the test to clear all mocks after each test? 😄 
```  afterEach(jest.clearAllMocks);```

Reply:
I have usually only added that when it is needed for the tests to run, but I can add it 🤗